PackageExplorer/Controls/PdbInfoViewer.xaml

```diff
@@ -35,7 +35,7 @@
             <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
 
-        <TextBox Text="Type:" Style="{StaticResource Key}" />
+        <TextBox Text="Type:" Style="{StaticResource Key}" IsTabStop="True" />
```

Comment:
This is a workaround to not auto focus the first cell in the `DataGrid` which would scroll it into view...

Reply:
Lets just add a comment in the code so we remember why we did that down the road :)

Reply:
Done.